### PR TITLE
Added validation when type of input is listvalue.

### PR DESCRIPTION
**Pull request by tawalisa**
Just a commit to fix https://github.com/graphql-java/graphql-java/issues/1793

### CODE REVIEW

File: src/test/groovy/graphql/validation/ValidationUtilTest.groovy [hunk 1]
```diff
@@ -88,13 +88,13 @@ class ValidationUtilTest extends Specification {
...
-        expect:
-        validationUtil.isValidLiteralValue(singleValue, type,schema)
-    }
+//    def "One value is a single element List"() {
+//        given:
+//        def singleValue = new BooleanValue(true)
+//        def type = list(GraphQLBoolean)
+//        expect:
+//        validationUtil.isValidLiteralValue(singleValue, type,schema)
+//    }
```
Comment (bbakerman) [inline src/test/groovy/graphql/validation/ValidationUtilTest.groovy]: A test is needed.  I think a pure unit test and perhaps a integration tests from the top

Comment (tawalisa) [reply src/test/groovy/graphql/validation/ValidationUtilTest.groovy]: added unit test. thanks.

**Comment by andimarek** [comment]
I am not sure ... If I remember correctly there was a rule in the spec about it that what we are doing is correct ... but I might be wrong.

**Comment by tawalisa** [comment]
Hi @andimarek 

I supposed you mentioned below spec. 
https://spec.graphql.org/June2018/#sec-Type-System.List

If the value is single value. I could not use [].  
But if the value is multiple values, it likes my case in https://github.com/graphql-java/graphql-java/issues/1793.   If it does not use []. it should be illegal. 
I have updated my change and add related test. Please let me know if any comments.

**Comment by tawalisa** [comment]
This is not an issue. Close it.

**Pull request closed without merging**